packages/overlay/src/ActiveOverlay.ts

```diff
@@ -245,18 +247,40 @@ export class ActiveOverlay extends SpectrumElement {
 
         this.state = 'active';
         this.feature();
-        if (this.placement && this.placement !== 'none') {
+        if (this.placement === 'none') {
```

Comment:
@davidyxu The code here should address the `100vh` situation causing scroll in your previous review, can you confirm that this has corrected the issue on your end (Android, right?) as well?

Reply:
👍 yep looks good on my end